public/css/main.css

```diff
@@ -219,3 +219,9 @@ body {
 }
 
 
+.graphColor {
+  color: white;
```

Comment:
Any reason for removing the transparency?

It would be good to keep the tooltip transparent like before so that it doesn't obscure the graph.